README.md

```diff
@@ -1,9 +1,54 @@
-[![Build Status](CI_BADGE_URL goes here)](REPO_URL goes here)
+Staging: https://ic-rails-huey-toby-staging.herokuapp.com/
```

Comment:
Minor improvement idea to center all this content and use badges for everything:

<p align="center">
  <img src="https://img.shields.io/github/actions/workflow/status/nimblehq/ic-rails-huey-toby/test.yml?branch=main" />
  <a href="https://ic-rails-huey-toby-staging.herokuapp.com/">
     <img src="https://img.shields.io/badge/Heroku-Staging-purple" />
  </a>
  <a href="https://ic-rails-huey-toby.herokuapp.com/">
     <img src="https://img.shields.io/badge/Heroku-Production-purple" />
  </a>
</p>